lib/fluent/plugin/in_tail.rb

```diff
@@ -370,17 +370,19 @@ def existence_path
     def refresh_watchers
       target_paths_hash = expand_paths
       existence_paths_hash = existence_path
-      
+
       log.debug {
         target_paths_str = target_paths_hash.collect { |key, target_info| target_info.path }.join(",")
         existence_paths_str = existence_paths_hash.collect { |key, target_info| target_info.path }.join(",")
         "tailing paths: target = #{target_paths_str} | existing = #{existence_paths_str}"
       }
 
-      unwatched_hash = existence_paths_hash.reject {|key, value| target_paths_hash.key?(key)}
+      @pf&.unwatch_removed_targets(target_paths_hash)
+
+      removed_hash = existence_paths_hash.reject {|key, value| target_paths_hash.key?(key)}
       added_hash = target_paths_hash.reject {|key, value| existence_paths_hash.key?(key)}
 
-      stop_watchers(unwatched_hash, immediate: false, unwatched: true) unless unwatched_hash.empty?
+      stop_watchers(removed_hash, immediate: false) unless removed_hash.empty?
```

Comment:
I'm not sure whether we can safely remove `unwatched` for `follow_inodes false` case.

Reply:
In `follow_inodes false` case it's safer not to change the behaviour for now.
The issue we currently want to solve is only the log duplication issue on `follow_inodes false` case.

Reply:
Thanks for your review!
I see!
I have limited the impact only to `follow_inodes true`!